src/app/components/PostsList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import axios from 'axios';
+import Post from './Post.js';
+
+export default class PostsList extends React.Component {
+  constructor (props) {
+    super (props);
+    this.state = {
+        posts: []
+    }
+    axios.get('https://jsonplaceholder.typicode.com/posts')
+    .then (response => {
+      this.setState = ({posts: response.data})
```

Comment:
```suggestion
      this.setState({posts: response.data})
```
this.setState() - не свойство, а метод объекта